gcloud-java-dns/src/test/java/com/google/gcloud/dns/it/ITDnsTest.java

```diff
@@ -147,12 +147,9 @@ private static void assertEqChangesIgnoreStatus(ChangeRequest expected, ChangeRe
   }
 
   private static void waitForChangeToComplete(String zoneName, String changeId) {
-    while (true) {
-      ChangeRequest changeRequest = DNS.getChangeRequest(zoneName, changeId,
-          Dns.ChangeRequestOption.fields(Dns.ChangeRequestField.STATUS));
-      if (ChangeRequest.Status.DONE.equals(changeRequest.status())) {
-        return;
-      }
+    ChangeRequest changeRequest = DNS.getChangeRequest(zoneName, changeId,
```

Comment:
Nit, though this is just a test you may want to consider overloading this method with a `ChangeRequest` param (this method can delegate to it) and using it in some of your test-cases that already have the change request instance.